src/terraform_gui.cpp

```diff
@@ -521,6 +521,7 @@ static void ResetLandscapeConfirmationCallback(Window *w, bool confirmed)
 		/* Now that all vehicles are gone, we can reset the engine pool. Maybe it reduces some NewGRF changing-mess */
 		EngineOverrideManager::ResetToCurrentNewGRFConfig();
 
+		InvalidateWindowData(WC_GAME_OPTIONS, WN_GAME_OPTIONS_AI);
```

Comment:
Seems random to have this here. What does this solve and why at this point?

Reply:
![screenshot 90](https://user-images.githubusercontent.com/43006711/51808885-374eee00-2292-11e9-8a1a-cfe61ec4b7d2.png)

It's for this. the AI's are then removed, so I have to update the window. If I dont update, it's gonna reflect them as being active, with the green smiles for a while, until I drag the window around.